spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -26,7 +26,10 @@ A full list of changes delivered in the 2.0 release can be found at link:https:/
 
 === Incompatible Changes
 
-- ConfigSource#getPropertyNames is no longer a default method. The implementation of a ConfigSource must implement this method. (link:https://github.com/eclipse/microprofile-config/issues/431[431])
+- ConfigSource#getPropertyNames is no longer a default method. The implementation of a ConfigSource must implement this method. (link:https://github.com/eclipse/microprofile-config/issues/431[431]
+- Previous versions of the specification would not evaluate Property Expressions. As such, previous working
+configuration may return behave differently (if the used configuration contains values with Property Expressions
+syntax). This new behaviour can be disabled. (link:property-expressions.asciidoc[Property Expressions])
```

Comment:
This new behaviour can be disabled. (link:property-expressions.asciidoc[Property Expressions]) =>
Check (link:property-expressions.asciidoc[Property Expressions]) on how to go back to the previous behaviour.